modules/core/src/smithy4s/schema/Schema.scala

```diff
@@ -180,6 +180,7 @@ object Schema {
   val boolean: Schema[Boolean] = Primitive.PBoolean.schema(prelude, "Boolean")
   val byte: Schema[Byte] = Primitive.PByte.schema(prelude, "Byte")
   val bytes: Schema[ByteArray] = Primitive.PBlob.schema(prelude, "Blob")
+  val blob: Schema[ByteArray] = Primitive.PBlob.schema(prelude, "Blob")
```

Comment:
question: aren't these the same? I would've thought you'd replace `ByteArray` with the new `Blob`.

Reply:
@daddykotex has jumped on a WIP branch  started. I should have made it clearer that I had further changes planned for this and didn't want to open right away.

I'm gonna turn this into a draft that I'll pick up later

Reply:
Well for some reason turning into draft doesn't work on phone